topics/galaxy-interface/tutorials/workflow-parameters/tutorial.md

```diff
@@ -222,6 +223,7 @@ In this example we will construct a workflow where we calculate the sum of all v
 
 With this you're ready to run the workflow!
 
+> <a name="running-the-workflow-2"></a>
```

Comment:
```suggestion
```

every hands on box has an automatic ID, please use that instead.

https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-parameters/tutorial.html#hands-on-running-the-workflow